src/components/Header/Header.tsx

```diff
@@ -0,0 +1,51 @@
+import { SetStateAction } from 'react';
+import cn from 'classnames';
+
+interface HeaderProps {
+  isAllCompleted: boolean;
+  handleSubmit: (event: React.FormEvent) => void;
+  setNewTodoTitle: React.Dispatch<SetStateAction<string>>;
```

Comment:
can we have () => void?
we should use useCallback for such things and probably React.memo